bindings/c/CMakeLists.txt

```diff
@@ -257,7 +257,6 @@ endif()
     add_test(NAME fdb_c_upgrade_single_threaded_630api
       COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
             --build-dir ${CMAKE_BINARY_DIR}
-            --disable-log-dump
```

Comment:
I'm lacking context. Could you explain why it was initially disabled? Might have been for a good reason.

Reply:
I had initially disabled the logs, because I was getting failures quite often while developing, and I could lookup the full logs from the file system. Now we observe sporadic errors in these tests, but they happen only in the Apple CI. The only way we get logs from there is to dump them to the console.